Makefile

```diff
@@ -98,7 +100,9 @@ container: manager
 docker-push:
 	docker push ${REGISTRY}/${IMG}:${IMAGE_TAG}
 	docker tag ${REGISTRY}/${IMG}:${IMAGE_TAG} ${REGISTRY}/${IMG}:${IMAGE_GIT_TAG}
+	docker tag ${REGISTRY}/${IMG}:${IMAGE_TAG} ${REGISTRY}/${IMG}:partial
```

Comment:
what is this used for?

Reply:
I used this to test at different cluster, we can remove it.